### PR TITLE
Add the rest of the content loaders logic and CSS

### PR DESCRIPTION
**Pull request by estacioneto**
#### What is the purpose of this pull request?

Add the rest of the content loaders logic and CSS

#### What problem is this solving?

Some content loaders weren't implemented in the previous version.

#### How should this be manually tested?

[Access the workspace](https://estacio--storecomponents.myvtex.com/google-pixel2/p), set the Network connection to **slow 3G** and click in some Related Product.

#### Screenshots or example usage
image

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by andreldsa — APPROVED** [review]

File: react/ProductDetails.js [hunk 1]
```diff
@@ -188,45 +192,44 @@ class ProductDetails extends Component {
...
-                {!Number.isNaN(
-                  +path(['AvailableQuantity'], this.commertialOffer)
-                ) &&
-                  (path(['AvailableQuantity'], this.commertialOffer) > 0 ? (
-                    <div>
-                      <div className="pv2">
-                        <BuyButton
-                          skuItems={[
+                {showBuyButton ? (
+                  <div>
```
Comment (lucasecdb) [inline react/ProductDetails.js]: Can we use a `Fragment` here?

File: react/ProductDetails.js [hunk 2]
```diff
@@ -188,45 +192,44 @@ class ProductDetails extends Component {
...
-                      <div className="pv4">
-                        {/* FIXME: Get this country data correctly */}
-                        <ShippingSimulator
-                          skuId={this.selectedItem.itemId}
-                          seller={this.sellerId}
-                          country="BRA"
-                        />
-                      </div>
+                          ]
+                        }>
```
Comment (lucasecdb) [inline react/ProductDetails.js:208]: I prefer the `>` to be in a new line.

**Review by vitoria — APPROVED** [review]

File: react/ProductDetails.js (hunk 2 above)
Comment (estacioneto) [reply react/ProductDetails.js:208]: prettier things :/

Comment (lucasecdb) [reply react/ProductDetails.js:208]: oh, ok then

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**